vocabularies/Common.xml

```diff
@@ -803,6 +803,11 @@ The minimum structure is
           <Annotation Term="Core.Description" String="Initial value, e.g. empty string, is a valid and significant value" />
         </Property>
       </ComplexType>
+      <ComplexType Name="ValueListRangeIn" BaseType="Common.ValueListParameterIn">
```

Comment:
The subtype is still called "range" but the functional scope is restricted to EQ, LE and GE. Are there plans to extend this in the future to CP and BT? If not, consider
- changing the type name
- whether a subtype is really needed.

One additional property whose default value reflects the current behavior does not necessitate a subtype. I asked for a subtype when it still had many more properties.